docs/contributors/code/native-mobile-integration-test-guide.md

```diff
@@ -0,0 +1,368 @@
+# Integration Test Guide
+
+## What’s an integration test?
+
+Integration testing is defined as a type of testing where different parts are tested as a group. In our case, the parts that we want to test are the different components that are required to be rendered for a specific block or editor logic. In the end, they are very similar to unit tests as they are run with the same command using the Jest library. The main difference is that for the integration tests, we’re going to use a specific library [`react-native-testing-library`](https://testing-library.com/docs/react-native-testing-library/intro/) for testing how the editor renders the different components.
+
+## Anatomy of an integration test
+
+A test can be structured with the following parts:
+
+-   Set up
+-   Rendering
+-   Query elements + Fire events
+-   Expect correct element behaviour
+-   Cleanup
+
+## Setup
```

Comment:
I'd use H3 (`###`) for the sections under `Anatomy of an integration test`, wdyt?

Reply:
I see that the following main sections like "Common flows" use H1 (`#`) so maybe for "Anatomy of an integration test" we could do the same and keep these sections with H2 (`##`).

Reply:
You're right that the section titles and subtitles have inconsistent sizes. I'm going to tweak it a little, because in my opinion only the document title itself should be H1 (`#`).